docker/iml-copy-settings.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
Can we rename this to `copy-embedded-settings`?

Reply:
Sure